packages/vscode-extension/src/handlers.ts

```diff
@@ -1134,41 +1133,14 @@ export async function validateLocalPrerequisitesHandler(): Promise<string | unde
  * Prompt window to let user install the app in Teams
  */
 export async function installAppInTeams(): Promise<string | undefined> {
-  let shouldContinue = false;
-  try {
-    let teamsAppId: string;
-    if (isV3Enabled()) {
-      teamsAppId = await commonUtils.getV3TeamsAppId(
-        globalVariables.workspaceUri!.fsPath,
-        environmentManager.getLocalEnvName()
-      );
-    } else {
-      const debugConfig = await commonUtils.getDebugConfig(
-        false,
-        environmentManager.getLocalEnvName()
-      );
-      if (debugConfig?.appId === undefined) {
-        throw new UserError(
-          ExtensionErrors.GetTeamsAppInstallationFailed,
-          ExtensionSource,
-          "Debug config not found"
-        );
-      }
-      teamsAppId = debugConfig.appId;
+  if (isV3Enabled()) {
```

Comment:
Could "v3 enabled" be removed as well?

Reply:
Can be removed with no risk. Just leave it to the one who will clean v3 feature flag :).